models-info/src/main/java/com/flipkart/fdp/ml/transformer/FillNAValuesTransformer.java

```diff
@@ -0,0 +1,51 @@
+package com.flipkart.fdp.ml.transformer;
+
+import com.flipkart.fdp.ml.modelinfo.FillNAValuesTransformerModelInfo;
+
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Transforms input/ predicts for a {@link FillNAValuesTransformerModelInfo} model representation
+ * captured by  {@link FillNAValuesTransformerModelInfo}.
+ */
+public class FillNAValuesTransformer implements Transformer {
+    private final FillNAValuesTransformerModelInfo modelInfo;
+
+    public FillNAValuesTransformer(FillNAValuesTransformerModelInfo modelInfo) {
+        this.modelInfo = modelInfo;
+    }
+
+    @Override
+    public void transform(Map<String, Object> input) {
+        for(Map.Entry<String, Object> entry : modelInfo.getNaValuesMap().entrySet()) {
+            if( isNA(input.get(entry.getKey()))) {
+                input.put(entry.getKey(), entry.getValue());
+            }
+        }
+    }
+
+    private boolean isNA(Object data) {
+        if( null == data) {
+            return true;
+        }
+        if( data instanceof Double) {
```

Comment:
Are there testcases to cover Double and Float isNan cases. This would be required to comprehensively test parity between pipeline and transformer.